core/gdxsv/gdxsv_update.cpp

```diff
@@ -330,6 +330,10 @@ std::shared_future<bool> GdxsvUpdate::StartSelfUpdate() {
 		auto current_version = std::string(GIT_VERSION).erase(0, 6);
 		auto trash_path = std::string(getenv("HOME")) + "/.Trash/" + GetFlycastFileNameWithVersion(current_version);
 		
+		while (file_exists(trash_path.c_str())) {
+			trash_path = trash_path.substr(0, trash_path.length() - 4) + "_.app";
+		}
```

Comment:
cannot nowide::remove(trash_path) ?

Reply:
Since macOS application is a folder, directory traversal and recursively delete its contents first is required to delete an application.
Another 1 liner solution would be calling `system("rm -rf {path}")` (seems too dangerous :P)